tests/Spectra.py

```diff
@@ -72,7 +83,7 @@ def tearDown(self):
         del self.maxRMS
 
     def testSpectrumConstructors(self):
-        if True:
+        if self.runTestSpectrumConstructors:
```

Comment:
Filed ticket PIPE2D-193 as changing the indentation of all lines will mess with git blame and I thought it would be better placed in a new ticket